frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
```

Comment:
I'll some general findings here (each as single conversations so that you can resolve them step by step):

URL: http://localhost:4200/settings/core/pipelines
Issue: Background of some text is white, not matching the light grey background.
Screenshot:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/7999bfd1-fa6b-4f55-8a22-a2d3371f1cae)